code/datums/flock/flock.dm

```diff
@@ -18,7 +18,9 @@ var/flock_signal_unleashed = FALSE
 	var/list/priority_tiles = list()
 	var/list/deconstruct_targets = list()
 	var/list/traces = list()
+	var/free_traces = 0
 	var/queued_trace_deaths = 0
+	var/max_trace_count = 0
```

Comment:
Code doc comments pls
don't worry about doing the rest of them, just the ones added here

Reply:
Added if looks okay